### PR TITLE
prevent errors on php 8.1

### PR DESCRIPTION
**Pull request by garak**
This should fix issue #79 
I found maybe a better way, that should keep compatibility even with PHP 7.3

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -22,7 +22,7 @@
         "ext-dom": "*",
         "monolog/monolog": "~1.7||~2.0",
         "phpunit/php-file-iterator": "~2.0||^3.0",
-        "symfony/console": "~3.4||~4.0||~5.0"
+        "symfony/console": "~3.4||~4.0||~5.0||~6.0"
```
Comment (garak) [inline composer.json:25]: This is not directly related, but anyway I think that being compatible with Symfony 6 can be nice

**Comment by mmoll** [comment]
merged, thanks @garak!

**Comment by garak** [comment]
> merged, thanks @garak!

Thank you. What about a new release? ;-)

**Comment by mmoll** [comment]
> What about a new release?

#81 needs to get solved first 🔥
